### DISCUSSION TITLE
How to update subfinder tool

### DISCUSSION THREAD

**Discussion by CyberBeta** [projectdiscovery/subfinder#367] [Q&A] [answered]
Hey Team,
I have old subfinder version. where when I go to config.yaml file I found some of recon application is not there. like: chaos, dnsdb, spyse, etc. So how can I update this. and I can set API keys with that.

**Comment by CyberBeta** [comment]
Any response please

**Comment by ehsandeep** [comment, marked as answer]
Hi @CyberBeta, 

If you install the latest version of the subfinder with  and remove 

1. Take backup for config file.
2. Install the latest version of subfinder `go install -v github.com/projectdiscovery/subfinder/v2/cmd/subfinder`
3. Remove the old config file with `rm $HOME/.config/subfinder/config.yaml`
4. Run subfinder and new config will be auto-created at `$HOME/.config/subfinder/config.yaml`
5. Update the config file with API keys
